Realm/Realm.Weaver/Analytics/Analytics.cs

```diff
@@ -338,12 +343,9 @@ private void AnalyzeClassMethods(TypeDefinition type)
                         continue;
                     }
 
-                    var index = new int[]
-                    {
-                        key.IndexOf("get_", StringComparison.Ordinal),
-                        key.IndexOf("set_", StringComparison.Ordinal),
-                        key.IndexOf("add_", StringComparison.Ordinal)
-                    }.Max();
+                    var (prefix, index) = prefixes.Select(p => (p, key.IndexOf(p, StringComparison.Ordinal)))
+                        .OrderByDescending(p => p.Item2)
+                        .First();
```

Comment:
Is this change for performance or readability?

Reply:
This is fairly nitty and possibly unnecessary, but it was triggering my OCD as the previous code was relying on all prefixes being exactly 4 characters long. I really dislike hardcoding values like that as it obscures the connection between the pieces of code using them (the index lookup and then the substring call).